wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -215,6 +216,7 @@ namespace :wagn do
           end
         end
         Card::Cache.reset_global
+        # FIXME: can this be associated with the machine module somehow?
```

Comment:
If it is only stuff like this one, and it is not common this may be ok, but what is the right solution to move this sort of logic to a module?  At least maybe move the code to the module and do something here like: if defined?(Card::Set::MachineSomething) { Card::Set::MachineSomething.cleanup }  

Better would be a more standard way of adding stuff to the config and iterating through railties to find the modules with a particular hook.